src/main/java/net/glowstone/block/blocktype/BlockRedstoneTorch.java

```diff
@@ -55,7 +55,7 @@ public void placeBlock(GlowPlayer player, GlowBlockState state, BlockFace face,
     @Override
     public void updatePhysics(GlowBlock me) {
```

Comment:
@mastercoms I can't really reproduce what you are saying, for me redstone mechanisms work pretty fine.. can you post a screenshot or something?